### ISSUE TITLE
Issue with  get_cansim_sqlite() giving an error message when collecting_and_normalizing()

### ISSUE THREAD

**Issue by chrisferris3** [mountainMath/cansim#110]
Hi @mountainMath.  I started having an issue with pulling cansim files using the cansim function **get_cansim_sqlite()** just before June 24, 2022.  I tried the suggested auto-refresh = TRUE, but for some reason I am still getting problems. I have trimmed my code to drop extraneous data and geography filters.

Here is my script:
```
#### SEPH - Monthly by Provinces ############## 
library(tidyverse)
library(cansim) 

CODR_TBL <- c("14-10-0371")

# Get employment data by industry
con <- get_cansim_sqlite(CODR_TBL, timeout = 3000, auto_refresh = TRUE)

# head(con)
# get_cansim_table_overview(CODR_TBL)
### collect and normalize the data.
a <- con %>% collect_and_normalize() 
```

For some reason, I keep getting an error message:

> Error in cansim_old_to_new(t) : 
>   Unable to match old CANSIM table number      NA
> In addition: Warning message:
> In mask$eval_all_filter(dots, env_filter) : NAs introduced by coercion

I checked if this issue arises for tables which had an old cansim number. It still occurs for me. For example, use:

`CODR_TBL <- c("14-10-0287")`

instead of the above table name. It also was crashing for me.

I am wondering, am I missing a required library (libraries), or perhaps something else?

- I use the Windows version of R. 
- I run the desktop version of R.
- I have updated to R version 4.2.1, and cansim version 0.3.11. 
- The libraries I have loaded by default are: (**base, datasets, graphics, grDevices, methods, stats, utils**). 
- I load the **cansim** library, and the **tidverse** libraries: (**dplyr, forcats, ggplot2, purrr, readr,  stringr,  tibble, tidyr**).

**Comment by mountainMath** [comment]
Ok, a bit of a shot in the dark, but I have been making some changes that might fix this. I can't test it because I don't have access to a windows machine right now, but can you try installing the v0.3.12 candidate and see if that fixes the problem for you @chrisferris3 ?
```
remotes::install_github("mountainmath/cansim@v0.3.12")
```

**Comment by chrisferris3** [comment]
I will give it a try right now and tell you how it goes.

**Comment by chrisferris3** [comment]
I restarted R, and then ran remotes::install_github("mountainmath/cansim@v0.3.12", force = TRUE).

Then I reran my script as above, and it worked! 

Thank you for making the adjustments!!

**Comment by mountainMath** [comment]
fixed in v0.3.12
